Example/Tests/PlayerEventTests.swift

```diff
@@ -129,12 +129,19 @@ class PlayerEventsSpec: QuickSpec {
                         haveBeenCalled(withArgs: ["newState": "\(PlayerState.CONVIVA_PAUSED.rawValue)"])
                     )
                 }
-
                 it("on stall started") {
                     playerDouble.fakeStallStartedEvent()
-                    expect(spy).to(
-                        haveBeenCalled(withArgs: ["newState": "\(PlayerState.CONVIVA_BUFFERING.rawValue)"])
-                    )
+                        expect(spy).toEventuallyNot(
```

Comment:
This test case may be replaced with following 2 test cases
- StallEnded event immediately after StallStarted and no player state change
- Stall Ended event with a gap of >= .10 seconds after StallStarted event and player state change to PlayerState.CONVIVA_BUFFERING expected here.

Reply:
Thanks Lucky, Unit tests added